snorkel/slicing/utils.py

```diff
@@ -25,13 +25,13 @@ def add_slice_labels(
         A DictDataLoader whose dataset.Y_dict attribute will be modified in place
     base_task
        The Task for which we want corresponding slice tasks/labels
-    slice_labels
+       S_matrix
```

Comment:
do we say `L_matrix` and `T_matrix`? or just `L` and `T`?

Reply:
I don't think there is a transformation matrix; but I'll change it to `S` for consistency with `L`